platform/android/app/src/main/kotlin/coop/polypoly/polypod/homescreen/HomeScreenFragment.kt

```diff
@@ -0,0 +1,775 @@
+                fontFamily = FontFamily(
+                    Font(tile.style.titleFont.family)
+                ),
+                fontSize = tile.style.titleFont.size,
+                lineHeight = tile.style.titleFont.lineHeight,
+                textAlign = tile.style.titleFont.alignment,
+            )
+        }
+    }
+}
+
+@Composable
+fun Footer(footer: Footer) {
+    val foregroundColor = if (isLight(footer.style.backgroundColor)) Color.Black else Color.White // ktlint-disable max-line-length
+    val buttonForegroundColor = if (isLight(footer.style.buttonBackgroundColor)) Color.Black else Color.White // ktlint-disable max-line-length
+    Card(
+        shape = RoundedCornerShape(footer.layout.cornerRadius)
+    ) {
+        Column(
+            verticalArrangement = Arrangement.spacedBy(
+                footer.layout.verticalSpacing
+            ),
+            modifier = Modifier
+                .background(footer.style.backgroundColor)
+                .padding(
+                    top = footer.layout.padding,
+                    start = footer.layout.padding,
+                    end = footer.layout.padding,
+                    bottom = footer.layout.padding
+                ),
+        ) {
+            Text(
+                text = footer.model.title,
+                color = foregroundColor,
+                overflow = TextOverflow.Ellipsis,
+                fontWeight = footer.style.titleFont.weight,
+                fontFamily = FontFamily(
+                    Font(footer.style.titleFont.family)
+                ),
+                fontSize = footer.style.titleFont.size,
+                lineHeight = footer.style.titleFont.lineHeight,
+                textAlign = footer.style.titleFont.alignment,
+            )
+            Text(
+                text = footer.model.description,
+                color = foregroundColor,
+                overflow = TextOverflow.Ellipsis,
+                fontWeight = footer.style.descriptionFont.weight,
+                fontFamily = FontFamily(
+                    Font(footer.style.descriptionFont.family)
+                ),
+                fontSize = footer.style.descriptionFont.size,
+                lineHeight = footer.style.descriptionFont.lineHeight,
+                textAlign = footer.style.descriptionFont.alignment,
+            )
+            Column(
+                verticalArrangement = Arrangement.spacedBy(
+                    footer.layout.verticalSpacing
+                ),
+                horizontalAlignment = Alignment.CenterHorizontally,
+                modifier = Modifier.fillMaxWidth()
+            ) {
+                Image(
+                    painter = painterResource(id = footer.model.imageId),
+                    contentDescription = null,
+                    contentScale = ContentScale.Fit,
+                    alignment = Alignment.Center
+                )
+                Button(
+                    onClick = { /*TODO*/ },
```

Comment:
Remember this TODO. Or say where and when...

Reply:
Will be handled separately, as we don't yet have the screen for LearnMore.